utils/src/lib.rs

```diff
@@ -179,20 +179,31 @@ pub enum ConfigParseError {
     InvalidValue(String),
 }
 
-/// parse members from string
+/// parse members from string like "node1=addr1,addr2,node2=add3,addr4,addr5,node3=addr6"
 /// # Errors
 /// Return error when pass wrong args
 #[inline]
-pub fn parse_members(s: &str) -> Result<HashMap<String, String>, ConfigParseError> {
+pub fn parse_members(s: &str) -> Result<HashMap<String, Vec<String>>, ConfigParseError> {
```

Comment:
Could we use `;` to split nodes like `node1=addr1,addr2;node2=add3,addr4,addr5;node3=addr6`, this makes it easier to read and easier to parse

Reply:
I found that using `;` to split these is not compatible with previous versions. This might lead us to modify configurations in many places (CI scripts, personal test environments, etc)